validator/gke_enable_stackdriver_kubernetes_engine_monitoring_test.rego

```diff
@@ -14,7 +14,7 @@
 # limitations under the License.
 #
 
-package templates.gcp.GCPGKEEnableStackdriverKubernetesEngineMonitoringConstraintV1
```

Comment:
Any particular reason to remove the `constraint` from the name?

Reply:
See commit descriptions